### DISCUSSION TITLE
Disable inheritance for specific field

### DISCUSSION THREAD

**Discussion by TS1997** [pimcore/pimcore#9446] [Q&A / Support]
Hello!

I'm currently building a webshop with lots of product variants using Pimcore. Most fields, such as texts and more, should be inheritable but there are a few fields such as the SKU and other variant specific values that must be required and unique. How would I go about disabling inheritance for these specific fields in order to make sure they stay unique? I'm using the latest Pimcore 10.0.4 if that matters

**Comment by NiklasBr** [comment]
There is a feature request for this: #5416

**Reply by TS1997** [reply]
I've added a comment there pushing for it
